relay-config/src/config.rs

```diff
@@ -524,13 +524,17 @@ struct SentryMetrics {
     ///
     /// Defaults to 5.
     pub meta_locations_max: usize,
+    /// Whether to override the [`received_at`] field in the [`BucketMetadata`] with the current
+    /// receive time of the instance.
+    pub override_received_at_metadata: bool,
 }
 
 impl Default for SentryMetrics {
     fn default() -> Self {
         Self {
             meta_locations_expiry: 15 * 24 * 60 * 60,
             meta_locations_max: 5,
+            override_received_at_metadata: false,
```

Comment:
By default we do not want to override the metadata to avoid unexpected behaviors due to the missing config param.